bigchaindb_driver/offchain.py

```diff
@@ -90,88 +89,83 @@ def prepare_transaction(*, operation='CREATE', owners_before=None,
 
-        * ``owners_after``, ``asset``, and ``inputs`` MUST be set.
+        * ``recipients``, ``asset``, and ``inputs`` MUST be set.
         * ``metadata`` MAY be set.
-        * The argument ``owners_before`` is ignored.
+        * The argument ``tx_signers`` is ignored.
 
     """
     operation = _normalize_operation(operation)
     return _prepare_transaction(
         operation,
-        owners_before=owners_before,
-        owners_after=owners_after,
+        tx_signers=tx_signers,
+        recipients=recipients,
         asset=asset,
         metadata=metadata,
         inputs=inputs,
     )
 
 
 def prepare_create_transaction(*,
-                               owners_before,
-                               owners_after=None,
+                               tx_signers,
+                               recipients=None,
                                asset=None,
                                metadata=None):
     """
     Prepares a ``"CREATE"`` transaction payload, ready to be
     fulfilled.
 
     Args:
-        owners_before (:obj:`list` | :obj:`tuple` | :obj:`str`): One
+        tx_signers (:obj:`list` | :obj:`tuple` | :obj:`str`): One
             or more public keys representing the issuer(s) of the asset
             being created.
-        owners_after (:obj:`list` | :obj:`tuple` | :obj:`str`, optional):
+        recipients (:obj:`list` | :obj:`tuple` | :obj:`str`, optional):
             One or more public keys representing the new owner(s) of the
             asset being created. Defaults to ``None``.
-        asset (:obj:`dict`, optional): The asset being created. Defaults
-            to ``None``.
+        asset (:obj:`dict`, optional): The metadata associated with the
+            asset being created with this transaction. Defaults to ``None``.
         metadata (:obj:`dict`, optional): Metadata associated with the
             transaction. Defaults to ``None``.
 
     Returns:
         dict: The prepared ``"CREATE"`` transaction.
 
-    .. important:: If ``owners_after`` is not given, or evaluates to
-        ``False``, it will be set equal to ``owners_before``::
+    .. important:: If ``recipients`` is not given, or evaluates to
+        ``False``, it will be set equal to ``tx_signers``::
 
-            if not owners_after:
-                owners_after = owners_before
+            if not recipients:
+                recipients = tx_signers
 
     """
-    if not isinstance(owners_before, (list, tuple)):
-        owners_before = [owners_before]
+    if not isinstance(tx_signers, (list, tuple)):
+        tx_signers = [tx_signers]
     # NOTE: Needed for the time being. See
     # https://github.com/bigchaindb/bigchaindb/issues/797
-    elif isinstance(owners_before, tuple):
-        owners_before = list(owners_before)
-
-    # TODO: This will only work for non-divisible. If its a divisible asset
-    # Transaction.create will raise an exception saying that divisible assets
-    # need to have amount > 1.
```

Comment:
This bit was removed because we don't declare an asset as being 'divisible' anymore; this is inferred from the conditions' amounts.